src/Server/SSH/SSHBind.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include <cstdint>
+#include <memory>
+#include <string>
+#include "base/types.h"
+
+struct ssh_bind_struct;
+using ssh_bind = ssh_bind_struct *;
+struct ssh_session_struct;
+using ssh_session = ssh_session_struct *;
```

Comment:
same comment as for `ssh_key` for confusing naming
also, it's not in CH code-style, should be something like `SshBindPtr`

Reply:
Fixed in `Remove unused get methods and fix types naming for ssh objects`